locations/__init__.py

```diff
@@ -1,5 +1,12 @@
+__version__ = "1.0"
+
 import requests_cache
 
-__version__ = "1.0"
+from locations.settings import REQUESTS_CACHE_BACKEND_SETTINGS, REQUESTS_CACHE_ENABLED
 
-requests_cache.install_cache(expire_after=60 * 60 * 24 * 3, backend="filesystem")
+try:
+    if REQUESTS_CACHE_ENABLED:
+        requests_cache.install_cache(**REQUESTS_CACHE_BACKEND_SETTINGS)
+except Exception as e:
+    print("requests_cache install failed:")
+    print(e)
```

Comment:
pre-commit is failing because of these prints. Can you add this file as an ignored file for the pre-commit check?